docs/source/guide/zne-3-options.myst

```diff
@@ -0,0 +1,371 @@
+Mitiq achieves this task by *digital* noise scaling, i.e., with circuit manipulations that indirectly increase the effect of noise.
+More details on digital ZNE can be found in [What is the theory behind ZNE?](zne-5-theory.myst)
+
+In Mitiq a noise scaling method is represented by a *noise scaling function* that takes as input a `circuit` and a real `scale_factor` and
+returns an `scaled_circuit`. For a noiseless backend, `scaled_circuit` has the same effect as `circuit`. For a noisy backend, 
+`scaled_circuit` is more sensitive to errors depending on the magnitude of `scale_factor`. 
+
+### Unitary Folding 
+
+Mitiq provides several noise scaling functions. Most of them are based on the repeated application of the *unitary folding* technique in
+which a unitary $G$ is mapped as follows:
+
+$$G \longrightarrow G G^\dagger G.$$
+
+If this is applied to individual gates of a `circuit`, we call it *local folding*. If $G$ is the entire `circuit`, we call it *global folding*.
+
+The Mitiq function for global folding is:
+
+- {func}`.fold_global()`.
+
+The Mitiq functions for local folding are: 
+- {func}`.fold_gates_at_random()`;
+- {func}`.fold_gates_from_left()`;
+- {func}`.fold_gates_from_right()`;
+- {func}`.fold_all()`.
+
+There are multiple functions for local folding since it can be applied to the gates of a circuit according to different orderings:
+at random, from left (starting from the initial gates), from right (starting from the final gates), etc.. 
+For more details on folding functions, we suggest to click on the functions listed above and check the associated API docs.
+
+If not specified by the user, the default noise scaling method in Mitiq is {func}`.fold_gates_at_random()`.
+
++++
+
+**Note:** *All folding functions can be applied to circuits defined in any supported frontend. For example, in the next code cells we use Cirq to represent quantum circuits.*
+
++++
+
+#### The special case of odd integer scale factors
+
++++
+
+Both local and global folding, if applied uniformly to all the gates of `circuit`, produce a `scaled_circuit` that has 3 times more gates than the input `circuit`.
+This corresponds to the `scale_factor=3` setting. For example:
+
+```{code-cell} ipython3
+import cirq
+from mitiq import zne
+
+# Get a circuit to fold
+qreg = cirq.LineQubit.range(2)
+circuit = cirq.Circuit(cirq.ops.H.on(qreg[0]), cirq.ops.CNOT.on(qreg[0], qreg[1]))
+print("Original circuit:", circuit, sep="\n")
+
+# Apply local folding
+scaled_circuit = zne.scaling.fold_gates_at_random(circuit, scale_factor=3)
+print("Locally folded circuit:", scaled_circuit, sep="\n")
+
+# Apply global folding
+scaled_circuit = zne.scaling.fold_global(circuit, scale_factor=3)
+print("Globally folded circuit:", scaled_circuit, sep="\n")
+```
+
+The same trick can be generalized to any odd integer `scale_factor`.
+In this case, folding functions apply the mapping $G \longrightarrow G (G^\dagger G)^ {(\text{scale_factor} - 1)/2}$. For example:
```

Comment:
```suggestion
In this case, folding functions apply the mapping $G \longrightarrow G (G^\dagger G)^ {(\text{scale_factor} - 1)/2}$. It can also be observed that when desired `scale_factor` is 1, the input circuit will remain unchanged. For example:
```

Reply:
Applied but in a different place: d3c6cee